Lesson_2js_homework/js/script.js

```diff
@@ -0,0 +1,52 @@
+// 1
+let yourName = prompt("Enter you name:");
+alert(`Привет, ${yourName}`);
+
+// 2
+let age = +prompt("Enter year of your birth:");
+const thisYear = 2021;
+alert(`Your age: ${thisYear - age}`);
```

Comment:
можно немного переименовать
```suggestion
let yearOfBirth = +prompt("Enter year of your birth:");
const thisYear = 2021;
alert(`Your age: ${thisYear - yearOfBirth}`);
```